task.sql

```diff
@@ -9,25 +9,49 @@ CREATE TABLE Countries (
     PRIMARY KEY (ID)
 );
 
-CREATE TABLE ProductInventory (
+CREATE TABLE Products (
     ID INT,
-    ProductName VARCHAR(50),
-    WarehouseAmount INT,
-    WarehouseName VARCHAR(50),
-    WarehouseAddress VARCHAR(50), 
-    CountryID INT,
-	FOREIGN KEY (CountryID) REFERENCES Countries(ID) ON DELETE NO ACTION,
+    Name VARCHAR(50),
     PRIMARY KEY (ID)
 );
 
+CREATE TABLE Warehouses (
+    ID INT,
+    Name VARCHAR(50),
+    Address VARCHAR(50), 
+    CountryID INT,
+    PRIMARY KEY (ID),
+    FOREIGN KEY (CountryID) REFERENCES Countries(ID) ON DELETE NO ACTION
+);
+
+CREATE TABLE ProductInventory (
+    ID INT,
+    ProductID INT,
+    WarehouseID INT,
+    Amount INT,
+    PRIMARY KEY (ID),
+	FOREIGN KEY (ProductID) REFERENCES Products(ID) ON DELETE NO ACTION,
+    FOREIGN KEY (WarehouseID) REFERENCES Warehouses(ID) ON DELETE NO ACTION
+);
+
 -- Populate test data
 
 INSERT INTO Countries (ID,Name)
-	VALUES (1, 'Country1');
-INSERT INTO Countries (ID,Name)
-	VALUES (2, 'Country2');
+	VALUES 
+    (1, 'Country1'),
+    (2, 'Country2');
+
+INSERT INTO Products (ID, Name)
+    VALUES
+    (1, 'Product1'),
+    (2, 'Product2');
+
+INSERT INTO Warehouses (ID,Name,Address,CountryID)
+	VALUES 
+    (1, 'Warehouse1', 'City1, Street1', 1),
+    (2, 'Warehouse2', 'City2, Street2', 2);
     
-INSERT INTO ProductInventory (ID,ProductName,WarehouseAmount,WarehouseName,WarehouseAddress,CountryID)
-	VALUES (1, 'AwersomeProduct', 2, 'Warehouse-1', 'City-1, Street-1',1);
-INSERT INTO ProductInventory (ID,ProductName,WarehouseAmount,WarehouseName,WarehouseAddress,CountryID)
-	VALUES (2, 'AwersomeProduct', 5, 'Warehouse-2', 'City-2, Street-2',2);
+INSERT INTO ProductInventory (ID,ProductID,WarehouseID,Amount)
+	VALUES 
+    (1, 1, 1, 1),
+    (2, 1, 2, 3);
```

Comment:
Fix EOF(end-of-file)
```suggestion
    (2, 1, 2, 3);
   
```
Each file should contain last empty line without any symbols and whitspaces.